docs/content-editing/creating-managing-and-reordering-blocks.md

```diff
@@ -0,0 +1,213 @@
+---
+myst:
+  html_meta:
+    "description": "Creating, managing, and reordering blocks"
+    "property=og:description": "Creating, managing, and reordering blocks"
+    "property=og:title": "Creating, managing, and reordering blocks"
+    "keywords": "Plone, Format, Block, Reorder"
+---
+# This guide offers a step-by-step approach to effectively create, manage, and reorder blocks within a Volto page.
+
+1.  In this example, you'll edit a page.
+
+    By default, the page includes a title block and the first text block.
+
+    To add another block, either press {kbd}`enter` or click on the {guilabel}`Add block` button <img alt="Add block icon" src="_static/circle-plus.svg" class="inline">.
+
+    ````{card}
+    ```{image} _static/creating-managing-and-reordering-blocks-01.jpeg
+    :alt: Click on the "+" to add another block
+    :target: _static/creating-managing-and-reordering-blocks-01.jpeg
```

Comment:
None of the images show up in https://plone-training--954.org.readthedocs.build/content-editing/creating-managing-and-reordering-blocks.html. I'll edit these, and leave the rest to you to fix.

`````suggestion
    To add another block, either press {kbd}`enter` or click on the {guilabel}`Add block` button <img alt="Add block icon" src="/content-editing/_static/circle-plus.svg" class="inline">.

    ````{card}
    ```{image} /content-editing/_static/creating-managing-and-reordering-blocks-01.jpeg
    :alt: Click on the "+" to add another block
    :target: /content-editing/_static/creating-managing-and-reordering-blocks-01.jpeg
`````